server/apps/main/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+
+class ShareExperienceForm(forms.Form):
+    experience = forms.CharField(label='Please share your story', max_length=500, strip=True,
+                                 widget=forms.Textarea(attrs={'placeholder':'Write your story here',
+                                                              'rows':'4',
+                                                              'class':'form-control'}))
+    experience.group = 1
+    wish_different = forms.CharField(label='Suggestions for what could have made your story better',
+                                     strip=True, max_length=500,
+                                     widget=forms.Textarea(attrs={'placeholder':'Write your story here',
+                                                              'rows':'4',
+                                                              'class':'form-control'}))
+    wish_different.group = 1
+    title = forms.CharField(label='Choose a title for your story', 
+                            strip=True,
+                            max_length=150,
+                            widget=forms.TextInput(attrs={'placeholder':'Title',
+                                                          'class':'form-control'}))
+    title.group = 1
+    
+    
+    
+    # add triggering here
+    abuse = forms.BooleanField(label = 'Abuse (physical, sexual, emotional and verbal)', required=False)
+    abuse.group = 2
+    violence = forms.BooleanField(label = 'Violence and Assualt', required=False)
+    violence.group = 2
+    drug = forms.BooleanField(label = 'Drug and/or Alcohol misuse', required=False)
+    drug.group = 2
+    mentalhealth = forms.BooleanField(label = 'Mental Health Issues', required=False)
+    mentalhealth.group = 2
+    negbody = forms.BooleanField(label = 'Negative body image', required=False)
+    negbody.group = 2
+    other = forms.CharField(label='',
```

Comment:
<img width="932" alt="Screenshot 2022-12-12 at 10 03 18" src="https://user-images.githubusercontent.com/674899/207017688-2a878ea2-31b0-482a-93ea-25e86c32d261.png">

I noticed is that the "other" category for data entering in the form doesn't have a checkbox attached to and I think it might be a result of how it's set up as a form field here. It's not a bit problem, but I think standard UI convention might be to have the checkbox regardless?

Reply:
I *think* the standard way of handling it is to have a checkbox regardless with a label *Other* and then have the free text box afterwards (see the image below for how Google Forms does it). 

<img width="615" alt="Screenshot 2022-12-12 at 13 05 38" src="https://user-images.githubusercontent.com/674899/207052433-0462991a-7bcf-4ec8-ae70-651bee2b207e.png">

I think this *might* be more accessible, as there is a clear label for the checkbox and text field if one does it like this. If you look at my screenshot above it becomes unclear what the text box relates to once you start typing as the 'example' text disppears, instead it then looks like the text box is related to mental health issues.

Reply:
Yes. I see exactly what you mean.